docs/contributing/testing.md

```diff
@@ -59,7 +59,7 @@ If you've created a new component, create a new `src/govuk/<COMPONENT>/<COMPONEN
 You should test that your contribution works:
 
 - in [Internet Explorer 8](https://frontend.design-system.service.gov.uk/supporting-ie8/), 9 and 10 - components do not need to look perfect
-- in [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in)
+- in [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in), including when you [resize text](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/resize-text-in-browsers.md)
```

Comment:
Just spotted, this URL doesn't need to include the GitHub part, it can just be

```suggestion
- in [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in), including when you [resize text](/docs/contributing/resize-text-in-browsers.md)
```

Reply:
@lfdebrux Ah cool, that's good to know - thanks!